scss/_dropdown.scss

```diff
@@ -37,26 +37,51 @@
 
 // The dropdown menu
 .dropdown-menu {
+  // scss-docs-start dropdown-css-vars
+  --#{$variable-prefix}dropdown-min-width: #{$dropdown-min-width};
+  --#{$variable-prefix}dropdown-padding: #{$dropdown-padding-y $dropdown-padding-y}; // Boosted mod
+  --#{$variable-prefix}dropdown-spacer: #{$dropdown-spacer};
+  @include rfs($dropdown-font-size, --#{$variable-prefix}dropdown-font-size);
+  --#{$variable-prefix}dropdown-color: #{$dropdown-color}; // stylelint-disable-line custom-property-empty-line-before
+  --#{$variable-prefix}dropdown-bg: #{$dropdown-bg};
+  --#{$variable-prefix}dropdown-border-color: #{$dropdown-border-color};
+  --#{$variable-prefix}dropdown-border-radius: #{$dropdown-border-radius};
+  --#{$variable-prefix}dropdown-border-width: #{$dropdown-border-width};
+  // Boosted mod: unused --#{$variable-prefix}dropdown-inner-border: #{$dropdown-inner-border-radius};
```

Comment:
I handled it like this because I've already submitted https://github.com/twbs/bootstrap/commit/2c5a16ff8bdffe64361c4e2b8ad2a2dd57c6fa14 that has been merged.